fundamentals/fundamentals-orphans/src/main/scala/izumi/fundamentals/orphans/OrphanDefs.scala

```diff
@@ -40,6 +40,21 @@ object `cats.Applicative` {
   @inline implicit final def catsApplicative: `cats.Applicative`[cats.Applicative] = null
 }
 
+final abstract class `cats.Monoid`[M[_]]
```

Comment:
```
`cats.kernel.Monoid`
```
is already available.

Reply:
ops, yeah, sorry, didn't spotted it.
will be fixed.

Reply:
I consider to add `LiftIO` and `MonadError`, but cats.effect.Resource also has CommutativeApplicative/Semigroup/SemigroupK

Also, they have a custom `MonadError` implementation, based on their underlying types. Do we need that too?

Reply:
@VladPodilnyk If you're willing to add the necessary methods, e.g. `Lifecycle.catchAll` you may add them.